src/main/java/seedu/duke/command/SeeAllEntriesCommand.java

```diff
@@ -43,11 +48,17 @@ public void execute(FinancialList list) {
         System.out.println("--------------------------------------------");
         String entryList = "";
         int entryCount = 0;
+        double cashflow = 0;
```

Comment:
Can consider having both an income and expense variable to allow all 3 items to be displayed.